src/api/index.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Root API abstract
+ *
+ * @author Vinh Le <lethanhvinh95@gmail.com>
+ *
+ */
+
+import axios from 'axios'
+
+let api: any
+let baseUrl: string = process.env.API_ENDPOINT
+	? process.env.API_ENDPOINT
```

Comment:
use https://www.npmjs.com/package/dotenv-webpack

remember to add .env file to gitignore